### PR TITLE
MOC: The Talent Cycle

### PR DESCRIPTION
**Pull request by MikeS-NZ**
MOC: Added five card scripts for spoiled MOM commander cards:

- Elspeth's Talent
- Liliana's Talent
- Rowan's Talent
- Teferi's Talent
- Vivien's Talent

### CODE REVIEW

File: forge-gui/res/cardsfolder/upcoming/teferis_talent.txt [hunk 1]
```diff
@@ -0,0 +1,12 @@
+Name:Teferi's Talent
+ManaCost:3 U U
+Types:Enchantment Aura
+K:Enchant planeswalker
+A:SP$ Attach | AILogic$ Pump | ValidTgts$ Planeswalker
+S:Mode$ Continuous | Affected$ Planeswalker.EnchantedBy | AddAbility$ TeferiUlt | Description$ Enchanted planeswalker has "[-8]: You get an emblem with 'You may activate loyalty abilities of planeswalkers you control on any player's turn any time you could cast an instant.'"
+SVar:TeferiUlt:AB$ Effect | Cost$ SubCounter<12/LOYALTY> | Planeswalker$ True | StaticAbilities$ InstantPlaneswalkers | Stackable$ False | Duration$ Permanent | AILogic$ Always | SpellDescription$ You get an emblem with "You may activate loyalty abilities of planeswalkers you control on any player's turn any time you could cast an instant."
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/teferis_talent.txt:7]: 12 = 8

Comment (MikeS-NZ) [reply forge-gui/res/cardsfolder/upcoming/teferis_talent.txt:7]: Copy paste got me there. Fixed, thanks!

**Comment by Hanmac** [comment]
imo all of them should need `DeckNeeds:Type$Planeswalker`

**Comment by MikeS-NZ** [comment]
> imo all of them should need `DeckNeeds:Type$Planeswalker`

One of them was missing this tag. Fixed, thanks!

File: forge-gui/res/cardsfolder/upcoming/teferis_talent.txt [hunk 2]
```diff
@@ -0,0 +1,12 @@
+Name:Teferi's Talent
+ManaCost:3 U U
+Types:Enchantment Aura
+K:Enchant planeswalker
+A:SP$ Attach | AILogic$ Pump | ValidTgts$ Planeswalker
+S:Mode$ Continuous | Affected$ Planeswalker.EnchantedBy | AddAbility$ TeferiUlt | Description$ Enchanted planeswalker has "[-12]: You get an emblem with 'You may activate loyalty abilities of planeswalkers you control on any player's turn any time you could cast an instant.'"
+SVar:TeferiUlt:AB$ Effect | Cost$ SubCounter<12/LOYALTY> | Planeswalker$ True | StaticAbilities$ InstantPlaneswalkers | Stackable$ False | Duration$ Permanent | AILogic$ Always | SpellDescription$ You get an emblem with "You may activate loyalty abilities of planeswalkers you control on any player's turn any time you could cast an instant."
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/teferis_talent.txt:7]: Needs emblem name

Comment (MikeS-NZ) [reply forge-gui/res/cardsfolder/upcoming/teferis_talent.txt]: Added name

**Review by tool4ever — APPROVED** [review]

**Pull request merged**